WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -929,8 +929,9 @@ private extension OrderDetailsViewController {
         }
 
         statusList.didSelectApply = { (selectedStatus) in
-            statusList.dismiss(animated: true) {
-                self.setOrderStatus(to: selectedStatus)
+            self.setOrderStatus(to: selectedStatus) {
+                statusList.configureRightButton()
```

Comment:
Could you explain why do you need to reconfigure the right button?

Reply:
You are right, I don't need to do this. I fixed it